src/validators/v1/default-rule.js

```diff
@@ -0,0 +1,129 @@
+const moment = require('moment')
+
+module.exports = {
+	create: (req) => {
+		try {
+			req.checkBody('type')
+				.isString()
+				.notEmpty()
+				.isIn(['session,mentor'])
+				.withMessage('Type is required and must be a string')
+
+			req.checkBody('target_field')
+				.isString()
+				.notEmpty()
+				.withMessage('Target field is required and must be a string')
+
+			req.checkBody('is_target_from_sessions_mentor')
+				.optional()
+				.isBoolean()
+				.withMessage('is_target_from_sessions_mentor must be a boolean')
+
+			req.checkBody('requester_field')
+				.isString()
+				.notEmpty()
+				.withMessage('Requester field is required and must be a string')
+
+			req.checkBody('field_configs').optional().isJSON().withMessage('Field configs must be a valid JSON')
+
+			req.checkBody('matching_operator')
+				.isString()
+				.notEmpty()
+				.isIn([
+					'equals',
+					'not equals',
+					'greater than',
+					'less than',
+					'greater than or equals',
+					'less than or equals',
+				])
+				.withMessage(
+					'Matching operator must be one of "equals", "not equals", "greater than", "less than", "greater than or equals", "less than or equals"'
+				)
```

Comment:
It should match from these:
	['equals', '='],
	['notEquals', '!='],
	['contains', '@>'],
	['containedBy', '<@'],
	['overlap', '&&'],
	['lessThan', '<'],
	['greaterThan', '>'],
	['lessThanOrEqual', '<='],
	['greaterThanOrEqual', '>='],
so please remove the spaces in between.